setup.py

```diff
@@ -9,10 +9,21 @@
 from setuptools.command.develop import develop
 from setuptools.command.install import install
 
-from owtf import __version__
 
 ROOT_DIR = os.path.dirname(os.path.realpath(__file__))
 
+#Extracts the version details 
+def get_version(init_path):
+    #Read the __init__ file
+    with open(init_path,"r+") as f:
+        init_page = f.read()
+    #Extract the version from the init file
+    for each_line in init_page.splitlines():
+        if each_line.startswith('__version__'):
+            delimiter = '"' if '"' in each_line else "'"
+            return(each_line.split(delimiter)[1])
```

Comment:
Modify the `return(<>)` to be `return <>`

Reply:
Thank You for your feedback, I will make the appropriate changes.